tasks/main.yml

```diff
@@ -92,6 +92,26 @@
       tags: configure
   when: inventory_hostname in openio_netdata_oio_hosts
 
+- name: Set open files limit non-systemd 
+  template:
+    src: "openio-security-limits.conf.j2"
+    dest: "/etc/security/limits.d/30-netdata.conf"
+    owner: "root"
+    group: "root"
+    mode: 0644
+  when: inventory_hostname in openio_netdata_oio_hosts
+  tags: configure
+
+- name: Set operating system limits on mmap  
+  template:
+    src: "openio-sysctl-netdata.conf.j2"
+    dest: "/etc/sysctl.d/netdata.conf"
+    owner: "root"
+    group: "root"
+    mode: 0644
+  when: inventory_hostname in openio_netdata_oio_hosts
```

Comment:
This needs to be applied everywhere, not only on OpenIO hosts

Reply:
Also, don't forget to reload sysctl when modifying the configuration. In fact, you can use sysctl module to set these parameters like this:

```
sysctl:
        sysctl_file: "/etc/sysctl.d/netdata.conf"
        name: "{{ item.key }}"
        value: "{{ item.value }}"
        sysctl_set: true
        reload: true
```